src/manifold/src/impl.cpp

```diff
@@ -496,8 +498,9 @@ void Manifold::Impl::CreateHalfedges(const Vec<ivec3>& triVerts) {
       if (h0.startVert != h1.endVert || h0.endVert != h1.startVert) break;
       if (halfedge_[NextHalfedge(pair0)].endVert ==
           halfedge_[NextHalfedge(pair1)].endVert) {
-        h0.face = -1;
-        h1.face = -1;
+        // FIXME: this actually does nothing
```

Comment:
This is an interesting one, I was trying to use a vector instead for this and was having weird issues. It turns out we declared `h0` and `h1` as `Halfedge`, so these assignments aren't doing anything. Changing them to `Halfedge&` breaks things, so I just commented them out for now.

Reply:
Ha, I think I put this in to try to debug something and then forgot about it. Let's remove.